src/query/plan/operator.cpp

```diff
@@ -4516,9 +4523,12 @@ void CallCustomProcedure(const std::string_view fully_qualified_procedure_name,
 class CallProcedureCursor : public Cursor {
   const CallProcedure *self_;
   UniqueCursorPtr input_cursor_;
-  mgp_result result_;
-  decltype(result_.rows.end()) result_row_it_{result_.rows.end()};
+  mgp_result *result_;
```

Comment:
Why are we switching to a pointer here?

Reply:
So that I can delete whole mgp_result with deallocating memory just.